test/JDBC/upgrade/latest/schedule

```diff
@@ -49,9 +49,8 @@ BABEL-3221
 BABEL-3204
 BABEL-3234
 BABEL-3402
-# Disabled until BABEL-3720 gets fixed
```

Comment:
Make sure to remove these tests from 14_6/schedule as well since 14_6/ folder was created from latest/

Reply:
not found in 14_6/schedule